databricks/koalas/frame.py

```diff
@@ -4250,7 +4265,7 @@ def to_spark(self, index_col: Optional[Union[str, List[str]]] = None):
 
     to_spark.__doc__ = SparkFrameMethods.__doc__
 
-    def to_pandas(self):
+    def to_pandas(self) -> "Pandas_DataFrame":
```

Comment:
I guess `-> pd.DataFrame` works?

Reply:
Could you also add it to `Series.to_pandas()`?

Reply:
Certainly! Updated.